simulation.py

```diff
@@ -139,7 +139,7 @@ def __init__(
                     to_log.append('existing ' + self.workingdir +
                                   ' renamed to ' + backupdir)
                     # make new empty working directory:
-                    mkdir(self.workingdir)
+                    mkdir(self.wSimulationorkingdir)
```

Comment:
Here is a small typo (probably a "wild paste" ;) ). wSimulationorkingdir -> workingdir